### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by TheRealTechWiz**
Add link to find latest version of libraries required. Would make getting started easier.

### CODE REVIEW

**Comment by mikepenz** [comment]
Oh we already do have a section about the latest releases here; 

image

thank you so much though for the PR, but I am afraid that it may not be needed.

**Comment by TheRealTechWiz** [comment]
The latest release provides only the latest version of materialDrawer. It does not tell about the latest versions of other libraries.
My commit adds links to external websites so I do understand your concern. Thank you for your time.

File: README.md [hunk 1]
```diff
@@ -73,6 +73,7 @@ implementation "com.mikepenz:materialdrawer-nav:${lastestMaterialDrawerRelease}"
 // Add for Android-Iconics support
 implementation "com.mikepenz:materialdrawer-iconics:${lastestMaterialDrawerRelease}"
 ```
+Visit [Google's Maven Repository](https://maven.google.com/web/index.html) or [MVN Repository](https://mvnrepository.com/artifact/com.mikepenz/materialdrawer) to find the latest version of the libraries.
```
Comment (mikepenz) [inline README.md:76]:
```suggestion
You can find dependency versions and all library releases on [MVN Repository](https://mvnrepository.com/artifact/com.mikepenz/materialdrawer).
```

**Comment by mikepenz** [comment]
I see. thank you very much for the addition. I've rephrased it a bit and changed to only link the mvn repo website for the dependencies.

**Comment by TheRealTechWiz** [comment]
Great Thankyou

**Pull request merged**